npm/src/controller/oauth.ts

```diff
@@ -1304,9 +1304,17 @@ export class OAuthController implements IOAuthController {
         if (!jwtSigningKeys || !isJWSKeyPairLoaded(jwtSigningKeys)) {
           throw new JacksonError(GENERIC_ERR_STRING, 500, 'JWT signing keys are not loaded');
         }
+        let profile = {};
+        if (this.opts.flattenRawClaims) {
+          profile = { ...codeVal.profile.claims.raw };
+          delete codeVal.profile.claims.raw;
```

Comment:
Won't this make the raw claims unavailable to userinfo in the downstream request ?

Reply:
Good spot.